unstructured_inference/visualize.py

```diff
@@ -0,0 +1,297 @@
+# Copyright (c) Megvii Inc. All rights reserved.
+# Unstructured modified the original source code found at
+# https://github.com/Megvii-BaseDetection/YOLOX/blob/ac379df3c97d1835ebd319afad0c031c36d03f36/yolox/utils/visualize.py
+
+import cv2
+import numpy as np
+
+__all__ = ["vis"]
+
+
+def vis(img, boxes, scores, cls_ids, conf=0.5, class_names=None):
```

Comment:
Can we call this function `draw_bounding_boxes` or similar? Since that better describe what the function is doing.

Reply:
Yes, the name was due the name of the original source, i'll just add a note of the renaming